runner/app/live/streamer/process.py

```diff
@@ -170,6 +170,7 @@ async def _initialize_pipeline(self):
                 return pipeline
         except Exception as e:
             self._report_error(f"Error loading pipeline: {e}")
+            logging.exception(e)
```

Comment:
tiny nit: passing in the exception is not needed here, since `logging.exception` already prints out the exception that's in-scope 🪄 

something like `logging.exception("Error loading pipeline")` should be perfectly fine.